index.html

```diff
@@ -97,14 +97,14 @@ <h3>Standard</h3>
                     <li><label class="checkbox" title="Attribute name must be lowercase."><input type="checkbox" id="attr-lowercase"> attr-lowercase</label></li>
                     <li><label class="checkbox" title="Attribute value must closed by double quotes."><input type="checkbox" id="attr-value-double-quotes"> attr-value-double-quotes</label></li>
                     <li><label class="checkbox" title="Attribute must set value."><input type="checkbox" id="attr-value-not-empty"> attr-value-not-empty</label></li>
-                    <li><label class="checkbox" title="Attribute name can not been duplication."><input type="checkbox" id="attr-no-duplication"> attr-no-duplication</label></li>
+                    <li><label class="checkbox" title="Attribute name can not been duplicated."><input type="checkbox" id="attr-no-duplication"> attr-no-duplication</label></li>
                     <li><label class="checkbox" title="Doctype must be first."><input type="checkbox" id="doctype-first"> doctype-first</label></li>
                     <li><label class="checkbox" title="Tag must be paired."><input type="checkbox" id="tag-pair"> tag-pair</label></li>
                     <li><label class="checkbox" title="The empty tag must closed by self."><input type="checkbox" id="tag-self-close"> tag-self-close</label></li>
                     <li><label class="checkbox" title="Special characters must be escaped."><input type="checkbox" id="spec-char-escape"> spec-char-escape</label></li>
                     <li><label class="checkbox" title="Id must be unique."><input type="checkbox" id="id-unique"> id-unique</label></li>
                     <li><label class="checkbox" title="Src of img(script,link) must set value."><input type="checkbox" id="src-not-empty"> src-not-empty</label></li>
-					<li><label class="checkbox" title="&lt;title&gt; must be present in &lt;head&gt; tag.."><input type="checkbox" id="title-require"> title-require</label></li>
+		    <li><label class="checkbox" title="&lt;title&gt; must be present in &lt;head&gt; tag.."><input type="checkbox" id="title-require"> title-require</label></li>
```

Comment:
Why this change?

Reply:
I was trying to make them line up, using the GitHub web interface so the tabs/spaces aren't quite aligned :-\

Reply:
try copy-pasting the whitespace from the line above instead, that might work better.